api/controllers/game/editmode.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (req, res) {
+  const { isRanked } = req.body;
+  const promiseGame = gameService.findGame({ gameId: req.session.game });
```

Comment:
We should make sure that this only fires when the user is already in a game by applying the `'isInGame'` policy to this endpoint in `config/policies.js`